dcs/container.py

```diff
@@ -32,21 +34,34 @@ class Container(ContainerBase):
 
     # outbound providers:
     dao_factory = get_constructor(MongoDbDaoFactory, config=config)
+    event_pub_provider = get_constructor(KafkaEventPublisher, config=config)
 
     # outbound translators:
     drs_object_dao = get_constructor(DrsObjectDaoConstructor, dao_factory=dao_factory)
+    event_publisher = get_constructor(
+        EventPubTranslator, config=config, provider=event_pub_provider
+    )
 
-    # outbound adapters:
-
+    # outbound adapters (not following the triple hexagonal translator/provider
```

Comment:
```suggestion
    # outbound adapters (not following the triple hexagonal translator/provider)
```

We should think about implementing a static code analys tool that looks for unclosed parenthesis in docs :-D